src/test/java/uk/gov/companieshouse/fixtures/EmailFixtures.java

```diff
@@ -4,9 +4,7 @@
 import uk.gov.companieshouse.model.dto.email.EmailDocument;
 import uk.gov.companieshouse.model.dto.email.SuccessfulPaymentEmailData;
 
-import static uk.gov.companieshouse.model.Constants.EMAIL_APP_ID;
-import static uk.gov.companieshouse.model.Constants.SUCCESSFUL_PAYMENT_EMAIL_SUBJECT;
-import static uk.gov.companieshouse.model.Constants.EMAIL_TOPIC;
+import static uk.gov.companieshouse.model.Constants.*;
```

Comment:
👀

Reply:
These are automatic IDE things

Reply:
I know it's a bit annoying. Can disable that following https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcards-for-class

Reply:
Ok the current default rule is 5 named imports or 3 static imports introduces a `*` to avoid files becoming 50% imports. I'll change it for this PR, but if we want this to be the standard then we should agree with the team and get it in the style guide.

Reply:
done